core/src/test/scala/org/apache/spark/scheduler/CoarseGrainedSchedulerBackendSuite.scala

```diff
@@ -180,7 +180,7 @@ class CoarseGrainedSchedulerBackendSuite extends SparkFunSuite with LocalSparkCo
     backend.driverEndpoint.askSync[Boolean](
       RegisterExecutor("3", mockEndpointRef, mockAddress.host, 1, logUrls, attributes, Map.empty))
 
-    sc.listenerBus.waitUntilEmpty(executorUpTimeout.toMillis)
+    sc.listenerBus.waitUntilEmpty()
```

Comment:
For my understanding, is this reducing the original timeout because it was 1 minutes before (`private val executorUpTimeout = 1.minute`)?

Reply:
Ah right. My bad that's 60000 ms. I'll just roll them back.

Reply:
Rolled back.